src/components/DonutPath.js

```diff
@@ -0,0 +1,65 @@
+const React = require('react');
+const d3 = require('d3');
+
+const DonutPath = React.createClass({
+  propTypes: {
+    arc: React.PropTypes.func,
+    color: React.PropTypes.string,
+    opacity: React.PropTypes.number,
+    value: React.PropTypes.object
```

Comment:
Can you provide explanations of these props? I also noticed there are no default props set. If that is the case, do we need to require all of the props?